css/style.css

```diff
@@ -1,19 +1,104 @@
 
- /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
-
 body {
     font-family: 'Roboto', sans-serif;
     -webkit-font-smoothing: antialiased;
+    font-style: normal;
+}
+
+#header {
```

Comment:
`#navbar` could be replaced with just `nav`  :) 
"nav" is our unique semantic element for Nav Bar.